### PR TITLE
Script interface

### PR DESCRIPTION
**Pull request by bootix**
Added a shell script based powermeter interface with usage example for Victron Multiplus II.

### CODE REVIEW

File: GetPowerFromVictronMultiplus.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#! /bin/sh
+
+# Script to read powermeter values from a Victron Multiplus II 
+# Needs "mbpoll" to be installed
```
Comment (reserve85) [inline GetPowerFromVictronMultiplus.sh:4]: can you please add the command in shell to do that?

**Comment by reserve85** [comment]
nice, can you add the changes to the changelist? and increase the version to 1.85 in config and *.py?

**Comment by bootix** [comment]
Done, hopefully it's what you where looking for.

**Comment by reserve85** [comment]
looks good, only the version in the config is missing. then i think its good!

**Pull request merged**